package.json

```diff
@@ -39,7 +39,7 @@
   },
   "activationEvents": [
     "onLanguage:solidity",
-    "workspaceContains:**/hardhat.config.ts"
+    "workspaceContains:**/hardhat.config.{ts,js}"
```

Comment:
Ha, good catch.

Do we need to add foundry.toml in here as well?

Reply:
Indeed! will add it